### ISSUE TITLE
Action fails to run because of broken dependency.

### ISSUE THREAD

**Issue by s0** [technote-space/toc-generator#192]
## Describe the bug: バグの概要

Currently facing an issue running this action, where it prints this error:

```
command [npx doctoc /home/runner/work/codeql-learninglab-actions/codeql-learninglab-actions/README*.md --title '**Table of Contents**' --github] exited with code 1. message: Command failed: npx doctoc /home/runner/work/codeql-learninglab-actions/codeql-learninglab-actions/README*.md --title '**Table of Contents**' --github
  internal/modules/cjs/loader.js:796
      throw err;
      ^
  
  Error: Cannot find module 'which'
  Require stack:
  - /usr/local/lib/node_modules/npm/node_modules/libnpx/index.js
  - /usr/local/lib/node_modules/npm/bin/npx-cli.js
      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
      at Function.Module._load (internal/modules/cjs/loader.js:686:27)
      at Module.require (internal/modules/cjs/loader.js:848:19)
      at require (internal/modules/cjs/helpers.js:74:18)
      at Object.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/libnpx/index.js:10:25)
      at Module._compile (internal/modules/cjs/loader.js:955:30)
      at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
      at Module.load (internal/modules/cjs/loader.js:811:32)
      at Function.Module._load (internal/modules/cjs/loader.js:723:14)
      at Module.require (internal/modules/cjs/loader.js:848:19) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [
  ```

See: https://github.com/github/codeql-learninglab-actions/runs/411835295?check_suite_focus=true

Looks like it's probably a dependency issue in `doctoc`, maybe some dependency relying on an implicit dependency?

## To Reproduce: 再現手順
1. Run a workflow with this action
1. See error

## Expected behavior: 期待する動作

Action runs without module resolution error.

## Operating environment: バグが発生した環境

Workflow with this config:

```yml
jobs:
  generateTOC:
    runs-on: ubuntu-latest
    name: Generate Table of Contents
    steps:
      - name: TOC Generator
        uses: technote-space/toc-generator@v2.0.6
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Seems to also happen when bumping to `v2.0.9`

**Comment by technote-space** [comment]
@s0 
I could not reproduce this `npx` in some of my repositories.
The "npx doctoc ..." command that was executed normally seems to have failed continuously from around 1/26.
From these things, it is possible that the "GitHub Actions" environment of this repository is broken.

The behavior using `npx` was introduced in `v2.0.6`.
https://github.com/technote-space/toc-generator/pull/178/files

Could you check if it works with `v2.0.5`?
If it works, I want to restore that behavior to the default.

**Comment by s0** [comment]
Seems to work with v2.0.5: https://github.com/github/codeql-learninglab-actions/runs/413994134?check_suite_focus=true

**Comment by technote-space** [comment]
@s0
Thank you for your cooperation.
I've released v2.1.0 fixes this issue.
